root/event/EventArt.js

```diff
@@ -0,0 +1,147 @@
+import RequestLogin from '../components/RequestLogin.js';
+import {SanitizedCatalystContext} from '../context.mjs';
+import EntityLink from '../static/scripts/common/components/EntityLink.js';
+import {commaOnlyListText}
+  from '../static/scripts/common/i18n/commaOnlyList.js';
+import {isAccountAdmin} from '../static/scripts/common/utility/privileges.js';
+
+import EventLayout from './EventLayout.js';
+
+type Props = {
+  +event: EventT,
+  +eventArt: $ReadOnlyArray<EventArtT>,
+};
+
+const EventArt = ({
+  eventArt,
+  event,
+}: Props): React$Element<typeof EventLayout> => {
+  const $c = React.useContext(SanitizedCatalystContext);
+  const title = lp('Event art', 'plural, header');
+
+  return (
+    <EventLayout entity={event} page="cover-art" title={title}>
+      <h2>
+        {event.event_art_presence === 'darkened'
+          ? lp('Cannot show event art', 'plural, header')
+          : title}
+      </h2>
+
+      {event.event_art_presence === 'darkened' ? (
+        <p>
+          {l(`Images for this item have been hidden
+              by the Internet Archive because of a takedown request.`)}
+        </p>
+      ) : eventArt.length ? (
+        <>
+          {eventArt.map(artwork => (
+            <div
+              className={
+                'artwork-cont' +
+                (artwork.editsPending ? ' mp' : '')
+              }
+              key={artwork.id}
+            >
+              <div className="artwork" style={{position: 'relative'}}>
+                <Artwork artwork={artwork} />
+              </div>
+              <p>
+                {l('Types:')}
+                {' '}
+                {artwork.types?.length ? (
+                  commaOnlyListText(artwork.types.map(
+                    type => lp_attributes(type, 'event_art_type'),
+                  ))
+                ) : lp('-', 'missing data')}
+              </p>
+              {artwork.comment ? (
+                <p>
+                  {artwork.comment}
+                </p>
+              ) : null}
+              <p className="small">
+                {l('All sizes:')}
+                {' '}
+                <ArtLinks artwork={artwork} />
+              </p>
+              {isAccountAdmin($c.user) ? (
+                <div className="buttons">
+                  <a
+                    href={'/event/' + event.gid +
+                          '/edit-event-art/' + artwork.id}
+                  >
+                    {lp('Edit', 'verb, interactive')}
+                  </a>
+                  <a
+                    href={'/event/' + event.gid +
+                          '/remove-event-art/' + artwork.id}
+                  >
+                    {l('Remove')}
+                  </a>
+                </div>
+              ) : null}
+            </div>
+          ))}
+
+          <p>
+            {exp.l(
+              `These images provided by the {eaa|Event Art Archive}.
+               You can also see them at the {ia|Internet Archive}.`,
```

Comment:
"provided by" is already passive :) But the change sounds good to me anyway.

Reply:
It could be either a nonverbal sentence (with the correct meaning and “provided” as an adjective), or a full sentence (with the opposite meaning and “provided” as a verb in the active form and the simple past tense).

But this is a weird in-between here as it starts with “These” which wouldn’t be needed for a nonverbal sentence.

Full sentences are usually expected. The second sentence is a full sentence too.